vspace/src/main/java/edu/asu/diging/vspace/core/aspects/previewDataAspect.java

```diff
@@ -0,0 +1,52 @@
+package edu.asu.diging.vspace.core.aspects;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.ui.Model;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.servlet.HandlerMapping;
+
+import edu.asu.diging.vspace.core.model.impl.Exhibition;
+import edu.asu.diging.vspace.core.services.IExhibitionManager;
+import edu.asu.diging.vspace.web.publicview.ExhibitionConstants;
+
+@Component
+@Aspect
+public class previewDataAspect {
```

Comment:
Java class names are never lowercased! Your Eclipse should have warned you, don't ignore that!